config/initializers/devise.rb

```diff
@@ -239,7 +239,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :facebook, ENV["FACEBOOK_APP_ID"], ENV["FACEBOOK_APP_SECRET"],
+  :info_fields => 'name,email', :image_size => 'square', :display => 'popup', :secure_image_url => 'true'
```

Comment:
Line is too long. [105/80]<br>Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.